internal/nginx/configurator_test.go

```diff
@@ -852,3 +851,29 @@ func TestUpdateEndpointsMergeableIngressFailsWithInvalidTemplate(t *testing.T) {
 		t.Errorf("UpdateEndpointsMergeableIngress returned \n%v, but expected \n%v", nil, "template execution error")
 	}
 }
+
+func TestGetNginxCommand(t *testing.T) {
```

Comment:
this test must be in `nginx_test.go`
you don't need to create a configurator, only the Controller